src/core/constants.js

```diff
@@ -100,7 +100,6 @@ export const SEARCH_SORT_TOP_RATED = 'rating';
 export const FETCH_ADDON = 'FETCH_ADDON';
 export const CATEGORIES_FETCH = 'CATEGORIES_FETCH';
 export const CATEGORIES_LOAD = 'CATEGORIES_LOAD';
-export const CATEGORIES_FAIL = 'CATEGORIES_FAIL';
```

Comment:
Thanks for removing these as you add errorHandlers. I admit I was slow to understand how they worked but I definitely like their flexibility more, even if they're a bit less obvious than just dispatching error actions. 👍